constraints.txt

```diff
@@ -11,11 +11,11 @@ MultiMapping==5.0
 Paste==3.10.1
 PasteDeploy==3.1.0
 Persistence==5.0
-RestrictedPython==7.1; python_version == '3.10'
-RestrictedPython==7.1; python_version == '3.11'
-RestrictedPython==7.1; python_version == '3.12'
-RestrictedPython==7.1; python_version == '3.8'
-RestrictedPython==7.1; python_version == '3.9'
+RestrictedPython==7.2; python_version == '3.10'
+RestrictedPython==7.2; python_version == '3.11'
+RestrictedPython==7.2; python_version == '3.12'
+RestrictedPython==7.2; python_version == '3.8'
+RestrictedPython==7.2; python_version == '3.9'
```

Comment:
My OCD made me twitch, seeing an alpha-only sort, instead of alphanumeric. More motivation to drop Python < 3.10 lol.

Reply:
This list is autogenerated from the information in `versions-prod.cfg` and `versions.cfg`. It is not maintained by hand, so this doesn't matter much.